duckduckgo/Android
# Add fire button and cache clearing

Asana Issue URL: 
https://app.asana.com/0/414730916066338/471193197990045

## Description
Shows fire button and allows cache and cookies to be cleared

## Steps to Test this PR:
TEST ONE
1. Login to a webpage e.g facebook
2. Press the fire button and select `Clear all data`
3. Open the webpage again ensure you are logged out

TEST TWO
1. Login to a webpage e.g facebook
2. Press the fire button and select `Cancel`
3. Ensure you are still logged in
